packages/ui/src/components/molecules/InfoBubble/InfoBubble.tsx

```diff
@@ -108,6 +115,15 @@ const getStyles = (badgeColor, Colors, isOpen) =>
       zIndex: 99,
       backgroundColor: Colors.backgroundColor,
     },
+    indicationCardRight: {
+      position: 'absolute',
+      right: Dimensions.get('window').width * 0.08,
+      paddingLeft: 10,
+      paddingVertical: 10,
+      paddingRight: 10,
+      zIndex: 99,
+      backgroundColor: Colors.backgroundColor,
+    },
```

Comment:
factoize common styles between the two side of indicator card in another style to simplify